crates/sui-kvstore/src/bigtable/proto/google.pem

```diff
@@ -0,0 +1,1128 @@
+# Operating CA: DigiCert
```

Comment:
Is this required to load their cert vs using normal root certs?

Reply:
Yeah, I checked a few reference implementations for Bigtable in Rust. Unfortunately, that’s what all of them ended up doing. We could potentially download it once on the fly when starting the process